docs/download-buttons.rst

```diff
@@ -38,40 +38,3 @@ Documentation
 See `helpers.py`_ for documentation and supported parameters for both buttons.
 
 .. _helpers.py: https://github.com/mozilla/bedrock/blob/main/bedrock/firefox/templatetags/helpers.py
-
-External referrers
```

Comment:
Note for reviewer: I'm removing this for now as we no longer want to encourage sites from using this, at least without some prior conversation around how they plan to handle data consent on their end. For AMO, legal have approved use here.